src/util/system.cpp

```diff
@@ -1128,10 +1128,22 @@ fs::path GetSpecialFolderPath(int nFolder, bool fCreate)
 void runCommand(const std::string& strCommand)
 {
     if (strCommand.empty()) return;
+    int nErr = 0;
 #ifndef WIN32
-    int nErr = ::system(strCommand.c_str());
+    nErr = ::system(strCommand.c_str());
 #else
-    int nErr = ::_wsystem(std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>,wchar_t>().from_bytes(strCommand).c_str());
+    STARTUPINFOA info;
+    PROCESS_INFORMATION processInfo;
+    ZeroMemory(&info, sizeof(info));
+    info.cb = sizeof(info);
+    ZeroMemory(&processInfo, sizeof(processInfo));
+    if (CreateProcessA(NULL, const_cast<char *>(strCommand.c_str()), NULL, NULL, FALSE, 0x08000000, NULL, NULL, &info, &processInfo)) {
+        WaitForSingleObject(processInfo.hProcess, INFINITE);
+        CloseHandle(processInfo.hProcess);
+        CloseHandle(processInfo.hThread);
```

Comment:
Where is `nErr` set to the process's return value?

Reply:
`nErr` is set only if the command did not succeed. Maybe it should be set in any case ?

Reply:
nErr is set only if the command did not succeed.

Reply:
`nErr` should always be set to the exit code of the process.

Reply:
I could have set `nErr` to `GetLastError()` outside of the `else` block but I've read from [the doc](https://msdn.microsoft.com/fr-fr/library/windows/desktop/ms679360(v=vs.85).aspx) that a windows process would not always return `0` on success : "some functions set the last-error code to 0 on success and others do not." This means that `if (nErr)` could pass and log as an error on windows even if the called process executed successfully.
I can change it (along with the log message as asked above) but I'll first give a look at using `ShellExecute` as Sipa proposed in #5834.